src/utils/paths.rs

```diff
@@ -26,8 +29,18 @@ pub fn get_app_home(app: &str) -> Result<PathBuf, Error> {
     let app_chains = get_app_chains_home()?;
     let app_home = app_chains.join(app);
 
-    // Creates the $app_home directory if not present
+    // Creates the `app_home` directory if not present
     fs::create_dir_all(&app_home)?;
 
     Ok(app_home)
 }
+
+pub fn get_celestia_home() -> Result<PathBuf, Error> {
```

Comment:
this can be moved to the celestia file, celestia is a specific DA layer, generic utils should have get_da_layer or generic names